### DISCUSSION TITLE
Unable to resolve type ItemsRepeater

### DISCUSSION THREAD

**Discussion by pyking-wang** [AvaloniaUI/Avalonia#11936] [Q&A] [answered]
error：AXN0002 XamlX.XamlParseException: Unable to resolve type ItemsRepeater from namespace https://github.com/avaloniaui .
I am using the latest preview version and I am not sure what the problem is.
![image](https://github.com/AvaloniaUI/Avalonia/assets/76192669/c8297db5-1f71-42b5-acb2-cb64ca08d904)

**Comment by timunie** [comment, marked as answer]
have you added the needed nuget package? Make sure to read also the breaking changes wiki. https://github.com/AvaloniaUI/Avalonia/wiki/Breaking-Changes#itemsrepeater-was-moved-to-its-own-separated-package

**Reply by pyking-wang** [reply]
Yes, I overlooked it.
Property Items have been changed to ItemsSource.
![image](https://github.com/AvaloniaUI/Avalonia/assets/76192669/e3db9c2b-2d68-41c1-9107-788de557cd7c)
Thank you, I will close it.

**Comment by KyleC69** [comment]
Documentation needs to reflect new changes with ItemsRepeater

**Reply by timunie** [reply]
Can you file a PR or at least open an issue in the docs repo?

**Comment by UsernameToSell** [comment]
I also have this problem.
I have installed the needed nuget-Package Avalonia.Controls.ItemsRepeater, yet I still get the error message:

`Unable to resolve type ItemsRepeater from namespace https://github.com/avaloniaui `

**Reply by timunie** [reply]
do you have a version miss-match maybe? All your Avalonia packages should be on the same version (also from 3rd party libs)

**Reply by UsernameToSell** [reply]
All my Avalonia packages are 11.0.4, except for the SVG packages, which has 11.0.0.1 as the highest version.

**Reply by timunie** [reply]
Please file a very minimal sample to reproduce. all 11.0 or higher should normally work imo.
